python/samples/kernel-syntax-examples/chat.py

```diff
@@ -19,7 +19,7 @@
 
 api_key, org_id = sk.openai_settings_from_dot_env()
 kernel.add_chat_service(
-    "chat-gpt", sk_oai.OpenAIChatCompletion("gpt-3.5-turbo", api_key, org_id)
+    "chat-gpt", sk_oai.OpenAIChatCompletion("gpt-4-0613", api_key, org_id)
```

Comment:
Was this change intentional? We'd like to keep this as gpt-3.5-turbo as it is more accessible than gpt-4.

Reply:
Sorry, I did not mean to commit that. Please reject that change.

Reply:
```suggestion
    "chat-gpt", sk_oai.OpenAIChatCompletion("gpt-3.5-turbo", api_key, org_id)
```